Shared/User.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  User.swift
+//  Fun Gen
+//
+//  Created by jules on 4/18/22.
+//
+
+import Foundation
+
+struct User: Codable {
+    let firstName: String
+    let lastName: String
```

Comment:
By asking for both first name and last name, we are potentially excluding people who might not have both. Do you think it's possible for us to just ask for their full name?

Reference: [Falsehoods Programmers Believe About Names](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)

Reply:
Went with 'username' instead. We can change this to 'Full Name', 'Name', or something else easily later.

Reply:
Sounds good